docs/api/baseplate/frameworks/queue_consumer/kombu.rst

```diff
@@ -0,0 +1,53 @@
+``baseplate.frameworks.queue_consumer.kombu``
+=============================================
+
+:doc:`Kombu <kombu:index>` is a library for interacting with queue brokers.
+
+This module provides a :py:class:`~baseplate.server.queue_consumer.QueueConsumerFactory`
+that allows you to run a :py:class:`~baseplate.server.queue_consumer.QueueConsumerServer`
+using Kombu that also integrates Baseplate's facilities into Kombu's message lifecycle.
+
+An abbreviated example of it in use::
+
+    def process_links(context, body, message):
```

Comment:
worth adding type annotations to this example so people know what they're getting?